### DISCUSSION TITLE
Test models and Julia builder for pysd

### DISCUSSION THREAD

**Discussion by rogersamso** [worlddynamics/WorldDynamics.jl#192] [Ideas]
Hi!

I see that you are trying to implement Vensim and Stella functions, an I thought you'd be interested to know about [this repo](https://github.com/SDXorg/test-models), to verify your implementations.

Also, I've been thinking about having a Julia builder for pysd for a long time (check [this](https://joss.theoj.org/papers/10.21105/joss.04329) out), and your work looks like an excellent starting point!

I'll keep an eye on your progress, and try to contribute soon.

Cheers

**Comment by natema** [comment]
Thanks Roger for contacting us!
We have indeed already interacted with PySD, especially regarding the Vensim DELAY function: see https://github.com/SDXorg/pysd/issues/391 and https://github.com/worlddynamics/WorldDynamics.jl/issues/183. 
As we point out in #183, the Vensim implementation of DELAYN is still unclear to us in some cases. Our implementation matches the one in Stella, so we hope the bug is on Vensim's side.
